packages/api/models/granules.js

```diff
@@ -378,6 +380,7 @@ class Granule extends Manager {
       timeToArchive: get(granule, 'post_to_cmr_duration', 0) / 1000,
       ...processingTimeInfo,
       ...temporalInfo,
+      cnm,
```

Comment:
I think `queryFields` is a very good idea, and DAACs can configure workflow to put anything there.

Reply:
It seems like we need to map `$.meta.cnm` to `$.meta.granule.queryFields.cnm` at the beginning and  `$.meta.cnmResponse` to `$.meta.granule.queryFields.cnm` so that we can save whatever minimal CNM info exists at the start of the workflow but also get all the info that doesn't exist until the end